HACKING.rst

```diff
@@ -0,0 +1,111 @@
+=======
+HACKING
+=======
+
+Getting Started
+---------------
+
+Getting the code
+~~~~~~~~~~~~~~~~
+
+If you just want to look at the code, you can just clone the main repository:
+
+::
+    $ git clone git@github.com:juju/theblues.git
+
+If you want to hack on theblues, you should follow the directions below.
+
+Setting up a development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Get Started!
+------------
+1. Fork_ the `theblues` repo on GitHub.
+
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/theblues.git
+
+3. Create a branch for local development::
+
+    $ git checkout -b name-of-your-bugfix-or-feature
+
+Now you can make your changes locally.
+
+4. When you're done making changes, check that your changes pass style and unit
+   tests, including testing other Python versions::
+
+    $ make check
+
+5. Commit your changes and push your branch to GitHub::
+
+    $ git add .
+    $ git commit -m "Your detailed description of your changes."
+    $ git push origin name-of-your-bugfix-or-feature
+
+6. Submit a pull request through the GitHub website.
+
+.. _Fork: https://github.com/juju/theblues/fork
```

Comment:
That's just the link for the first step in the section; ReST generally suggests putting all the links at the end of a block.